Documentation/upgrade.md

```diff
@@ -178,9 +193,9 @@ so we will delete the old pod and start the new toolbox.
 kubectl -n rook delete pod rook-tools
 ```
 After verifying the old tools pod has terminated, start the new toolbox. You will need to either create the toolbox using the yaml in the master branch
-or simply set the version of the container to `rook/rook:v0.7.0-27.gbfc8ec6` before creating the toolbox.
+or simply set the version of the container to `rook/toolbox:master` before creating the toolbox.
```

Comment:
i'm not sure on this one because the toolbox also has a bunch of general purpose stuff like `fio` and `gdb`.  I'm wondering if we'll keep the toolbox as the all-in-one debugging solution across all backends?

Reply:
although not entirely sure how that would work from a build perspective

Reply:
I have gone ahead and renamed the toolbox image to `rook/ceph-toolbox`.